pom.xml

```diff
@@ -29,9 +30,9 @@
     <artifactId>forgerock-parent</artifactId>
     <version>2.0.7</version>
```

Comment:
oh... actually... maybe you meant I made the 2.0.7 change on the 2.0.6 branch, in which case, yes, that's wrong. i'll fix it -- don't merge this just yet.